### PR TITLE
feat(EU Covid Certificate): [IAGP-35] Certificate in wrong format

### PR DESCRIPTION
**Pull request by debiff**
## Short description
This PR add the screen that inform the user that the selected certificate is in the wrong format (response code 400)

## List of changes proposed in this pull request
- Added the certificate not found screen (`EuCovidCertWrongFormatKoScreen.tsx`)

## How to test
Run `EuCovidCertNotFoundKoScreen.test.tsx ` with Jest.
On dev server `src/routers/features/eu_covid_cert/index.ts` return 400

### CODE REVIEW

File: locales/en/index.yml
Comment (thisisjp) [inline locales/en/index.yml:2472]:
```suggestion
        title: "This certificate cannot be \nretrieved: contact support"
```

File: locales/en/index.yml
Comment (thisisjp) [inline locales/en/index.yml:2462]:
```suggestion
        title: "It seems that the certificate does not \nexist: contact support"
```

**Review by thisisjp — CHANGES_REQUESTED** [review]
Not sure about the correct position of the line break, as I’ve tweaked a little bit the wording

File: locales/en/index.yml
Comment (debiff) [reply locales/en/index.yml:2469]: Here the result:

File: locales/en/index.yml
Comment (debiff) [reply locales/en/index.yml:2479]: Here the result:

File: ts/features/euCovidCert/screens/ko/EuCovidCertWrongFormatKoScreen.tsx
Comment (fabriziofff) [inline ts/features/euCovidCert/screens/ko/EuCovidCertWrongFormatKoScreen.tsx:12]: Could you add also the red version of this image?
According to the design, the image must be red

**Review by fabriziofff — CHANGES_REQUESTED** [review]

File: ts/features/euCovidCert/screens/ko/EuCovidCertWrongFormatKoScreen.tsx
Comment (debiff) [reply ts/features/euCovidCert/screens/ko/EuCovidCertWrongFormatKoScreen.tsx:12]: No problem for me, I used the other version in agreement with @thisisjp.

Comment (debiff) [reply ts/features/euCovidCert/screens/ko/EuCovidCertWrongFormatKoScreen.tsx]: Modified in [78e5b84](https://github.com/pagopa/io-app/pull/3093/commits/78e5b84850ee7b87a300e8e9e8938f41f6e6f918)

**Review by fabriziofff — APPROVED** [review]

**Comment by fabriziofff** [comment]
lgfm, waiting for @thisisjp approval

**Review by thisisjp — APPROVED** [review]

**Pull request merged**
